lib/rules/selector-max-compound-selectors/index.js

```diff
@@ -23,29 +24,21 @@ function rule(max) {
 	return (root, result) => {
 		const validOptions = validateOptions(result, ruleName, {
 			actual: max,
-			possible: [
-				// TODO: Issue #4985
-				// eslint-disable-next-line no-shadow
-				function (max) {
-					return typeof max === 'number' && max > 0;
```

Comment:
[note] This does not allow zero, but I think no problems if using `isNonNegative()` as with the other rules.

Reply:
A util for this is a great idea.

In the future (bug fix pull request), we could rename to `isNonNegativeInteger`, and we add [`isInteger()`](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Number/isInteger) to the conditional? For these rules, a maximum of a float, e.g. `1.5`, doesn't make sense.

Reply:
Renaming to `isNonNegativeInteger()` is a good idea! 👍 

Since we support Node 10, we need a polyfill of `isInteger()` still. But the polyfill is simple, so I'll do it with this PR. 💪 

https://github.com/stylelint/stylelint/blob/1e81784e3abcd6151e98c5cffc94703aec499d87/package.json#L196-L198

Reply:
Renamed. 944fab6a